RealmSwift/RealmApp.swift

```diff
@@ -0,0 +1,143 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import Foundation
+import Realm
+import Realm.Private
+
+/**
+An object representing the Realm App configuration
+
+- see: `RLMAppConfiguration`
+*/
+public typealias AppConfiguration = RLMAppConfiguration
+
+public typealias OptionalErrorCompletionBlock = ((Error?) -> Void)
+
+/// The `RealmApp` has the fundamental set of methods for communicating with a Realm
+/// application backend.
+
+/// This interface provides access to login and authentication.
+public class RealmApp {
```

Comment:
This also looks like it could just be a typealias?

Reply:
I do not want to expose any RLM prefixed elements to a swift user

Reply:
Moved to using a type alias instead